README.md

```diff
@@ -1,27 +1,28 @@
-# ejs-loader for webpack
+# ejs-loader for webpack with ECMAScript Modules
 
-[![npm](https://img.shields.io/npm/v/ejs-loader.svg)](https://www.npmjs.com/package/ejs-loader)
-[![Build Status](https://travis-ci.com/difelice/ejs-loader.svg)](https://travis-ci.com/difelice/ejs-loader)
+[![npm](https://img.shields.io/npm/v/@groveco/ejs-loader.svg)](https://www.npmjs.com/package/@groveco/ejs-loader)
 
-EJS (Underscore/LoDash Templates) loader for [webpack](http://webpack.github.io/). Uses [lodash template](http://lodash.com/docs#template) function to compile templates.
+EJS (Underscore/LoDash Templates) loader for [webpack](http://webpack.github.io/) compiled as an ECMAScript module. Uses [lodash template](http://lodash.com/docs#template) function to compile templates.
 
 If you are looking for the loader which uses [EJS templating engine](https://github.com/tj/ejs), there is [ejs-compiled-loader](https://github.com/bazilio91/ejs-compiled-loader)
 
 ## Installation
 
-`npm install ejs-loader`
+`npm install @groveco/ejs-loader`
+
 
 ## Usage
 
 [Documentation: Using loaders](http://webpack.github.io/docs/using-loaders.html)
 
 ``` javascript
-var template = require("ejs!./file.ejs");
+import template from '@groveco/ejs-loader?variable=data!./file.ejs';
 // => returns the template function compiled with undesrcore (lodash) templating engine.
 
 // And then use it somewhere in your code
 template(data) // Pass object with data
 ```
+The variable option is `required` to compile EJS templates into ES compatible modules. If the `variable` option is not provided as a loader or `query` [option](https://webpack.js.org/concepts/loaders/#loader-features), an `UnsupportedConfiguration` Exception will be thrown throw. Please see https://github.com/lodash/lodash/issues/3709#issuecomment-375898111 for additional details 
```

Comment:
If the option is required, why don't we give it a default instead of throwing an exception?

Reply:
@racoleman only reason I am opting for the exception here is it might not be clear how the default value plays into a given template. I want it to be clear that a value must be explicitly passed in and generic templates will not be supported, as opposed to the template being compiled without errors but at runtime syntax errors are potentially thrown.